crates/cubecl-wgpu/src/lib.rs

```diff
@@ -33,13 +34,14 @@ mod tests {
 }
 
 #[cfg(all(test, feature = "spirv"))]
+#[allow(unexpected_cfgs)]
 mod tests_spirv {
     pub type TestRuntime = crate::WgpuRuntime<crate::spirv::VkSpirvCompiler>;
     use cubecl_core::flex32;
     use half::f16;
 
     cubecl_core::testgen_all!(f32: [f16, flex32, f32, f64], i32: [i8, i16, i32, i64], u32: [u8, u16, u32, u64]);
-    cubecl_linalg::testgen_matmul_plane!([f16, flex32, f32]);
+    //cubecl_linalg::testgen_matmul_plane!([f16, flex32, f32]);
```

Comment:
Was this commented out on purpose?

Reply:
yes, they're very slow (on the order of half an hour) and should only be run when necessary